src/swagger-specs/audit-query.yaml

```diff
@@ -355,7 +355,7 @@ components:
             sandboxName: prod
             region: VA7
             requestId: hArqS4CAa8wfRPnKuxV4yaA82atxwzYu
-            authId: 80b7d887-9338-4cd5-9d79-2483b03f0160
+            authId: some-authid
```

Comment:
Same as my comment with the other example. It is odd to replace hard value example with a "some-authid" but if that is the feedback then thats fine

Reply:
I replaced this value because the gitleaks part of the validator was flagging this authId, but agreed - I'll look for some better wording than `some-authid`